content/styles/01-atoms/buttons.css

```diff
@@ -14,7 +14,7 @@ input[type=submit] {
   background-color: var(--color-primary);
   border: 1px solid var(--color-primary);
   border-radius: .3rem;
-  transition: color,background-color .3s ease;
+  transition: all .3s ease;
```

Comment:
I think we had selected which transition to apply as loading was doing weird stuff with padding and sizing buttons.
`all` may have unpredictable behavior as we don't control what is being transitioned.
No big deal and open to discuss about that.

Reply:
you rigth, I fix with b45a00a8c9fb4be8370db0d403ac1d7d4e26e1ce